docs/user/dashboard/lens-percentage.asciidoc

```diff
@@ -0,0 +1,71 @@
+
+In this guide we'll use the **flights** sample data sets to explore the different options we have to answer the following questions:
+
+* How are values distributed within my data?
+* What is the proportion of X within my data?
+* Is the distribution changing over time?
+
+=== Computing the proportion for the overall data
+
+The first step to understand the distribution is to have an overview for the overall data: this type of visualization is useful to compared multiple values to the whole dataset and understand their proportions.
+
+*Lens* offers different types of visualization in order to help with this type of question: *Pie* chart, *Donut* chart, *Treemap* and *Percentage* charts.
+
+In this example we'll investigate what are the most common reasons of flights delay within the dataset: the idea is to count all the available records, then group them by the type of delay recorded.
+
+To explain this visually, we can start from a *Pie* chart in Lens and drop the *Records* field into the `Size by` dimension, then drag and drop the *FlightDelayType* field into the `Slice by` dimension:
++
+[role="screenshot"]
+image::images/lens_percentage_1.png[Pie chart distribution]
+
+From the results it is possible to see that the majority of flights have had *No delay* at all, while the remaining flights had delay for several different reasons. One of these possible reasons has the label *Other*, which represents additional reasons not fetched yet by our configuration.
+In order to expand more what other reasons there are within the *Other* category, we can open the *Top values of FlightDelayType* configuration and increment the *Number of values* to 10:
++
+[role="screenshot"]
+image::images/lens_percentage_2.png[Pie chart distribution with all delay types]
+
+Notice how the suggestion system is also recommending other types of visualization for the same dataset, as the *Treemap* one. Clicking on the *Treemap* thumbnail an alternative representation of the proportion is now visualized on the main chart area:
++
+[role="screenshot"]
+image::images/lens_percentage_3.png[Treemap distribution]
+
+*Lens* lets you experiment with different types of chart, trying to keep the focus on the same type of question or similar. When not possible a warning sign will be shown in the Chart switcher.
+
+You can change now the chart type to *Bar vertical percentage* to have another alternative representation of the proportion:
++
+[role="screenshot"]
+image::images/lens_percentage_4.png[Percentage chart distribution]
+
+In the *Percentage* charts the *Break down by* configuration tells *Lens* how to split the value computed on the Vertical Axis, therefore the *Top values of FlightDelayType* configuration is moved there when changing chart.
```

Comment:
```suggestion
Since *Break down by* tells *Lens* how to split the value computed on the vertical axis, the *Top values of FlightDelayType* moves to the vertical axis when you change the chart type.
```